scripts/make_wheel.sh

```diff
@@ -264,6 +264,7 @@ package_wheel() {
   else
       NEW_WHEEL_PATH=${WHEEL_PATH/-py2-/-cp27-}
   fi
+  NEW_WHEEL_PATH=${WHEEL_PATH/linux/manylinux1}
```

Comment:
I think this should be:
```
NEW_WHEEL_PATH=${NEW_WHEEL_PATH/linux/manylinux1}
```

Since we already assigned `NEW_WHEEL_PATH` above and made some modifications to it.